.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
-pytest-selenium/* @openstax/unified-integration-testing
-
 * @openstax/rex
+
+pytest-selenium/* @openstax/unified-integration-testing
```

Comment:
On https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
I've read:
```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
```
I think it should be changed to: `pytest-selenium/ @openstax/unified-integration-testing`

I also can see that `unified-integration-testing` is nested inside `unified` team but I don't think it matters